.eslintignore

```diff
@@ -28,6 +28,8 @@ packages/gatsby/gatsby-admin-public
 packages/gatsby-codemods/transforms
 packages/gatsby-source-graphql/batching
 packages/gatsby-plugin-gatsby-cloud/components
+packages/gatsby-plugin-gatsby-cloud/context
+packages/gatsby-plugin-gatsby-cloud/models
```

Comment:
I was getting linting errors in these folders so added them to the ignore list as well